src/server/api/routes/orders.router.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const ordersController = require('../controllers/orders.controller');
+
+/**
+ * @swagger
+ * /orders:
+ *  get:
+ *    tags:
+ *    - Orders
+ *    summary: Get all orders
+ *    description:
+ *      Will return all orders.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  ordersController
+    .getOrders()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /orders/{ID}:
+ *  get:
+ *    tags:
+ *    - Orders
+ *    summary: Get order by ID
+ *    description:
+ *      Will return single order with a matching ID.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: ID
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The ID of the order to get
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ *      400:
+ *        description: Bad request. Category ID must be an integer and larger than 0.
+ *      404:
+ *        description: A category with the specified ID was not found.
+ */
```

Comment:
Same here should be order id

Reply:
👍🏻

Reply:
Ok now after our discussion and that we agreed you need to get orders by USER_ID this should be changed again. 
Please read carefully documentation you provide so it is logical.